### ISSUE TITLE
Build error on mac M1 with the simulator

### ISSUE THREAD

**Issue by jfougere** [adjust/cordova_sdk#160]
Using version 4.29.1 of this cordova plugin, we are getting the below error when trying to run on the simulator on an M1 mac:

```
ld: in Ubigi/Plugins/com.adjust.sdk/AdjustSdk.framework/AdjustSdk(ADJPackageParams.o), building for iOS Simulator, but linking in object file (Ubigi/Plugins/com.adjust.sdk/AdjustSdk.framework/AdjustSdk(ADJPackageParams.o)) built for iOS, file 'Ubigi/Plugins/com.adjust.sdk/AdjustSdk.framework/AdjustSdk' for architecture arm64
```

**Comment by Aditi3** [comment]
Hi @jfougere, We’re currently investigating this issue at our end and working on finding a meaningful solution. 

In the meantime, as a temporary workaround, could you exclude arm64 for the simulator architecture both from your project and the Pod project to resolve the error.

**Comment by Aditi3** [comment]
Fixed as of [v4.30.0](https://github.com/adjust/cordova_sdk/releases/tag/v4.30.0).

**Comment by Aditi3** [comment]
If needed and still an issue with the latest SDK version, feel free to comment/reopen.
